src/common.ts

```diff
@@ -170,7 +169,10 @@ export class LoggingCommon {
     // not sure if its correct but for now we always set it even if it has
     // nothing in it
     data.metadata =
-        this.inspectMetadata ? mapValues(metadata, util.inspect) : metadata;
+        this.inspectMetadata ? Object.keys(metadata).reduce((acc, key) => {
```

Comment:
Please extract this into a function. What is the tangible savings from removing `lodash.mapvalues` here?

Reply:
It seemed like a good idea at the time 😆 Would you rather me break it out into it's own method, or just add `lodash.mapvalues` back?  I am ok with either.

Reply:
My vote is to add it back, unless we have a reason not to.